.github/workflows/build.yml

```diff
@@ -3,7 +3,7 @@ name: Build and Upload
 on:
   push:
     tags:
-      - v[1].[0-9]+.[0-9]+
+      - v[0-9].[0-9]+.[0-9]+
```

Comment:
`version-hardcode.sh` won't work properly with this so if we release anything above v1 the binary may not report the correct version